.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -154,6 +161,10 @@ module backend_service {
   dataset_submissions_bucket = local.dataset_submissions_bucket
   datasets_bucket            = local.datasets_bucket
   execution_role             = local.ecs_execution_role
+
+  # Bump health_check_interval from 15 seconds to 30 seconds so that WMG snapshot download,
+  # which at the time of this writing is around 27GB, has time to complete.
+  health_check_interval      = 30 
```

Comment:
is 30s enough to download 27GBs?

Should we bump this to 60s to be safe? Having a long interval doesn't seem like it should matter much. Although if the task dies, it means that it would need to wait a little bit longer before restarting the task, because i think a failing health check is what triggers that.. Not sure - I'd like @ebezzi's input here.

Alternatively, is there a parameter that specifies a delay before the health check starts pinging in 15s intervals? That way we can add a one minute delay before health check starts and then ping at the standard 15s afterwards.

Reply:
We tested this and this is a fairly conservative interval. Increasing this has the side effect that deployments will generally take longer, so I would only do that if we observe failures. We have enough logs to understand, for instance, if the download took longer on that specific instance (but it's an internal AWS download, so the speed should be somewhat predictable).

There is no parameter to specify a delay, as far as I know.

Reply:
@atarashansky - 30 seconds was tested and has never failed so far. Health check fails if there are 10 consecutive pings to port 80 that fails where each ping is done 30 seconds apart. So as long as the server is can respond to the ping in 300 seconds (10 x 30 seconds), it should be good.

Meanwhile, observing the logs, I see that we are getting at a minimum 150 MiB/s and typically 220 MiB/s when downloading. So download of 27GB takes no longer than 180 seconds (27000/150).

I am not against incrementing the healthcheck interval if you and @ebezzi feel that is still cutting it too close

Reply:
300s is perfect - I wasn't aware of the 10 consecutive pings required. Resolving this thread!